.swiftlint.yml

```diff
@@ -1,16 +1,17 @@
-# last tested/reviewed with SwiftLint 0.45.1
+# last tested/reviewed with SwiftLint 0.50.3
 
 opt_in_rules:
+  - accessibility_label_for_image
```

Comment:
Some rules are irrelevent here (e.g, because they apply to SwiftUI only). I mostly keep this file as a codified changelog of SwiftLint additions, adding all opt-in rules by default, and removing those that get in the way (unless I disagree with it based on its name/summary).